js/foam/u2/Element.js

```diff
@@ -524,11 +541,29 @@ CLASS({
     },
 
     function cls2_(oldClass, newClass) {
-      if ( oldClass === newClass ) return;
-      this.removeCls(oldClass);
-      if ( newClass ) {
-        this.classes[newClass] = true;
-        this.onSetCls(newClass, true);
+      var old = {};
```

Comment:
cls2() should only handle a single class.  The parser should break multiple classes delimited with spaces into multiple calls to cls2().